indium-chrome.el

```diff
@@ -66,20 +72,21 @@ Open URL if provided."
                                       nil
                                       'indium-chrome-url-history
                                       (car indium-chrome-url-history))))
-  (make-process :name "indium-chrome-process"
-                :command (list (indium-chrome--find-executable)
-                               (format "--remote-debugging-port=%s" indium-chrome-port)
-                               (or url "")))
+
+  (let ((executable (indium-chrome--find-executable)))
+    (if executable
+        (make-process :name "indium-chrome-process"
+                      :command (list executable
+                                     (format "--remote-debugging-port=%s" indium-chrome-port)
+                                     (or url "")))
+      (user-error "Cannot find chrome/chromium binary (%s) in PATH" indium-chrome-executable)))
```

Comment:
Is there a reason why you moved the error handling here?

Reply:
No reason now. I'll move it back.

Reply:
Done.